### ISSUE TITLE
Behavior score doesn't match communication score

### ISSUE THREAD

**Issue by AlanKhamdeyev** [ValveSoftware/Dota2-Gameplay#13870]
### Description

Communication score went down to 5990 and behavior score is on 7981 mark. 
It doesn't match each other and communication score doesn't go up for like 60 matches straight
I had 5990 60 matches ago and have the same now. 
I can't communicate at all (i mean i can't use voice or text chat in game, so i can't be reported or get a like for communication).
Profile ID: 899646831

### Screenshots

![photo_2023-11-28_01-48-30](https://github.com/ValveSoftware/Dota2-Gameplay/assets/117160815/ada7db17-acb8-4957-9a98-bd2a35047d28)

**Comment by NAlexP** [comment]
> Yep, made like 5 posts about it here, other people been making posts about it for weeks before I had the issue. They dont give a fuck about it. The communication score is bugged, after it drops under 6000 it gets stuck at last value.

FALSE! STOP SPREADING NONSENSE! MINE IS STUCK TO THE BEHAVIOUR SCORE AT 895!

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/30490675/920a0ab2-0b63-4f00-8622-c26a794de6a9)

**Comment by legacy1234** [comment]
@NAlexP Its not nonsense. Not everyone has it bugged, the people who have it bugged its when it dropped for first time under 6000. Your score is not split in 2, you still have same values. I dont have same values and my comm score got stuck when it went under 6k. Behaviour score is still increasing/decreasing. Same case as the person who posted.
